### PR TITLE
doc: update mirroring guide to include pool/image status commands

### PR DESCRIPTION
**Pull request by trociny**
Fixes: http://tracker.ceph.com/issues/15746

### CODE REVIEW

**Comment by dillaman** [comment]
Minor: the "Fixes: " keyword should be the URL back to the ticket.

File: doc/rbd/rbd-mirroring.rst [hunk 1]
```diff
@@ -228,6 +228,34 @@ For example::
    local cluster's ``rbd-mirror`` daemon process is responsible for performing
    the resync asynchronously.
 
+Mirror Status
+=============
+
+For every mirroring image the ``rbd-mirror`` daemon stores its current status
```
Comment (dillaman) [inline doc/rbd/rbd-mirroring.rst]: "The peer cluster replication status is stored for every primary mirrored image. This status can be retrieved .."

File: doc/rbd/rbd-mirroring.rst [hunk 2]
```diff
@@ -228,6 +228,34 @@ For example::
...
+in the RBD mirror image directory. The mirroring status can be retrieved using
+the ``mirror image status`` and ``mirror pool status`` commands.
+
+To get status for an image, specify the ``mirror image status`` command along
```
Comment (dillaman) [inline doc/rbd/rbd-mirroring.rst]: Minor (just to keep it consistent with the rest of the doc): "To request the mirror image status with ``rbd``, specify the ``mirror image status`` command along with the pool and image name::"

File: doc/rbd/rbd-mirroring.rst [hunk 3]
```diff
@@ -228,6 +228,34 @@ For example::
...
+with the pool and image name::
+
+        rbd mirror image status {pool-name}/{image-name}
+
+For example::
+
+        rbd mirror image status image-pool/image-1
+
+To get summary status for a pool, specify  the ``mirror pool status`` command along
```
Comment (dillaman) [inline doc/rbd/rbd-mirroring.rst]: Same here

**Comment by trociny** [comment]
@dillaman Thanks, fixed.

File: doc/rbd/rbd-mirroring.rst [hunk 4]
```diff
@@ -228,6 +228,34 @@ For example::
...
 
+Mirror Status
+=============
+
+The peer cluster replication status is stored for every primary mirrored image.
+This status can be retrieved using the ``mirror image status`` and
+``mirror pool status`` commands.
+
+To request the mirror image status with ``rbd``, specify the
+ ``mirror image status`` command along with the pool and image name::
```
Comment (dillaman) [inline doc/rbd/rbd-mirroring.rst]: Minor: extra space causes the doc generator to move this down to the next line.

Comment (trociny) [inline doc/rbd/rbd-mirroring.rst]: @dillaman thanks, fixed.

**Pull request merged**